bin/inference/pycbc_inference

```diff
@@ -208,6 +217,11 @@ with ctx:
     for key,val in static_args.iteritems():
         try:
             static_args[key] = float(val)
+            continue
+        except:
+            pass
+        try:
```

Comment:
Maybe this should be nested or something. Its going to check if it can be typecast as a float and then a list of strings which isn't possible if the first is true.